scripts/jinx.lic

```diff
@@ -468,15 +468,16 @@ module Jinx
         return CLI.help()
       end
       # repo list
-      if argv.repo.eql?(true) && argv.list
+      if argv.repo.eql?(true) && argv.list.eql?(true)
```

Comment:
this ensures that these arguments are passed naked, instead of as flags, which makes the CLI more specific

example:
```
repo --list=taters # should fail, but currently doesnt
repo list               # will match this exactly going forward
```